Pearson-Advance/frontend-app-xtreme-labs
# PADV-1837 feat: create initial template for xtreme labs project

## Description

This PR remove unnecessary files and set the principal configuration to the MFE, to make it run in the port 2004, to the Xtreme Labs integration.

## How to test

1. Clone the repository in the ``src`` folder, using the branch PADV-1837
2. Inside folder repository, run the command ``npm install``, to install the required dependencies
3. After the installation, run the command ``npm start``, to run the MFE
4. It's necessary that your environment is running, otherwise you should see a black page
5. Go to ``http://{lms-base}:2004``, you should see a ``Xtreme Labs Project`` message, it indicates the MFE is working well

## Jira issue

- https://agile-jira.pearson.com/browse/PADV-1837
